moveit_ros/planning_interface/move_group_interface/src/move_group_interface.cpp

```diff
@@ -806,7 +806,7 @@ class MoveGroupInterface::MoveGroupInterfaceImpl
     if (code != rclcpp_action::ResultCode::SUCCEEDED)
     {
       RCLCPP_ERROR_STREAM(LOGGER, "MoveGroupInterface::plan() failed or timeout reached");
-      return false;
+      return res->error_code;
```

Comment:
Really minor nitpick: why not use `code` here since somebody took the trouble to define it?

```suggestion
      return code;
```

Reply:
Ah, I see why now. Will revert my change  :+1: